spec/index.bs

```diff
@@ -1457,60 +1456,61 @@ interface GPUBufferUsage {
+
+                    Issue(gpuweb/gpuweb#605): Explain that the resulting error buffer can still be mapped at creation.
+
+                    Issue(gpuweb/gpuweb#605): Explain what are a {{GPUDevice}}'s `[[allowed buffer usages]]`.
+                </div>
 
-        1. Let |b| be a new {{GPUBuffer}} object.
-        1. Set |b|.{{GPUBuffer/[[size]]}} to |descriptor|.{{GPUBufferDescriptor/size}}.
-        1. Set |b|.{{GPUBuffer/[[usage]]}} to |descriptor|.{{GPUBufferDescriptor/usage}}.
-        1. If |descriptor|.{{GPUBufferDescriptor/mappedAtCreation}} is `true`:
+            1. Let |b| be a new {{GPUBuffer}} object.
+            1. Set |b|.{{GPUBuffer/[[size]]}} to |descriptor|.{{GPUBufferDescriptor/size}}.
+            1. Set |b|.{{GPUBuffer/[[usage]]}} to |descriptor|.{{GPUBufferDescriptor/usage}}.
+            1. If |descriptor|.{{GPUBufferDescriptor/mappedAtCreation}} is `true`:
 
-            1. Set |b|.{{GPUBuffer/[[mapping]]}} to a new {{ArrayBuffer}} of size |b|.{{GPUBuffer/[[size]]}}.
-            1. Set |b|.{{GPUBuffer/[[mapping_range]]}} to `[0, descriptor.size]`.
-            1. Set |b|.{{GPUBuffer/[[mapped_ranges]]}} to `[]`.
-            1. Set |b|.{{GPUBuffer/[[state]]}} to [=buffer state/mapped at creation=].
+                1. Set |b|.{{GPUBuffer/[[mapping]]}} to a new {{ArrayBuffer}} of size |b|.{{GPUBuffer/[[size]]}}.
+                1. Set |b|.{{GPUBuffer/[[mapping_range]]}} to `[0, descriptor.size]`.
+                1. Set |b|.{{GPUBuffer/[[mapped_ranges]]}} to `[]`.
+                1. Set |b|.{{GPUBuffer/[[state]]}} to [=buffer state/mapped at creation=].
 
-            Else:
+                Else:
 
-            1. Set |b|.{{GPUBuffer/[[mapping]]}} to `null`.
-            1. Set |b|.{{GPUBuffer/[[mapping_range]]}} to `null`.
-            1. Set |b|.{{GPUBuffer/[[mapped_ranges]]}} to `null`.
-            1. Set |b|.{{GPUBuffer/[[state]]}} to [=buffer state/unmapped=].
+                1. Set |b|.{{GPUBuffer/[[mapping]]}} to `null`.
+                1. Set |b|.{{GPUBuffer/[[mapping_range]]}} to `null`.
+                1. Set |b|.{{GPUBuffer/[[mapped_ranges]]}} to `null`.
+                1. Set |b|.{{GPUBuffer/[[state]]}} to [=buffer state/unmapped=].
 
-        1. Set each byte of |b|'s allocation to zero.
-        1. Return |b|.
+            1. Set each byte of |b|'s allocation to zero.
+            1. Return |b|.
 
-        Note: it is valid to set {{GPUBufferDescriptor/mappedAtCreation}} to true without {{GPUBufferUsage/MAP_READ}}
-        or {{GPUBufferUsage/MAP_WRITE}} in {{GPUBufferDescriptor/usage}}. This can be used to set the buffer's
-        initial data.
+            Note: it is valid to set {{GPUBufferDescriptor/mappedAtCreation}} to true without {{GPUBufferUsage/MAP_READ}}
+            or {{GPUBufferUsage/MAP_WRITE}} in {{GPUBufferDescriptor/usage}}. This can be used to set the buffer's
+            initial data.
 
-    </div>
+        </div>
```

Comment:
is this the desired indentation? looks like 8 spaces here

Reply:
Yeah, this is consistent with the rest of the `<dl>` blocks and prevents a (minor) style issue.